[Doc]--Add example of offloading data to NFS (#12777)
apache/pulsar: site2/docs/tiered-storage-filesystem.md

## Tutorial

This section provides step-by-step instructions on how to use the filesystem offloader to move data from Pulsar to Hadoop Distributed File System (HDFS) or Network File system (NFS).

> <!--DOCUSAURUS_CODE_TABS-->

Reviewer: Have you previewed the content using `yarn start`?

<!--HDFS-->

To move data from Pulsar to HDFS, follow these steps.

> ### Step1: Prepare the HDFS environment

Reviewer: ```suggestion
### Step 1: Prepare the HDFS environment
```

This tutorial sets up a Hadoop single node cluster and uses Hadoop 3.2.1.

> **Tip**
>

> > For more information about setting up a Hadoop single node cluster, see [here](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/SingleCluster.html).

Reviewer: ```suggestion
> For how to set up a Hadoop single node cluster, see [here](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/SingleCluster.html).
```

How about changing the description more concise and direct?
